config.lua.dist

```diff
@@ -83,6 +83,7 @@ showOnlineStatusInCharlist = false
 yellMinimumLevel = 2
 yellAlwaysAllowPremium = false
 forceMonsterTypesOnLoad = true
+cleanProtectedZones = false
```

Comment:
it's called "protection zone", so this should probably be worded `cleanProtectionZones`.

Reply:
This is still not updated.